index.bs

```diff
@@ -134,6 +134,13 @@ This default audio session is represented as an {{AudioSession}} object that is
 
 # Extensions to the `Navigator` interface # {#extensions-to-navigator}
 
+Each {{Window}} has an <dfn>associated AudioSession</dfn>, which is an {{AudioSession}} object.
+It represents the default audio session that is used by the user agent to automatically set up the audio session parameters.
+The user agent will request or abandon audio focus when audio session [=audio session/elements=] start or finish playing.
```

Comment:
Do we want to force the implementers to follow this guideline? If so, should we use `MUST request or abandon audio focus`?

Reply:
It is a general description of the intent of the spec. The algorithms will precisely describe how UA is expected to implement this so that all UAs will behave the same.
I do not think that this sentence is precise enough to warrant a MUST.